chats/views.py

```diff
@@ -20,7 +27,16 @@ def chat_view(request):
             message.author = request.user
             message.group = chat_group
             message.save()
-            return redirect("chats:home")
-    return render(
-        request, "chats/chat.html", {"chat_messages": chat_messages, "form": form}
-    )
+            # return redirect('chats:home')
+            return redirect('chats:show', id=chat_group.id)
+    return render(request, "chats/chat_group.html", {'chat_messages': chat_messages, 'form': form})
+
+def chat_view_create(request):
```

Comment:
```suggestion
def chat_view_create(request):
```
可以命名為 `chat_create` 就好